projects/angular-calendar/src/date-adapters/dayjs/index.ts

```diff
@@ -0,0 +1,52 @@
+import { adapterFactory as baseAdapterFactory } from 'calendar-utils/date-adapters/moment';
```

Comment:
I don't think we even need a specific adapter for dayjs do we? As the API is identical to moment's we can just use that one and keep it simple?

Reply:
True. I added it as it looked like the convention but indeed DayJs uses the same API as Moment for those functions.
Maybe if this changes in the future we can readd it.